spark-bigquery-pushdown/pushdown_common_src/test/scala/com/google/cloud/spark/bigquery/pushdowns/SparkExpressionConverterSuite.scala

```diff
@@ -30,9 +34,19 @@ class SparkExpressionConverterSuite extends AnyFunSuite with BeforeAndAfter {
   private val schoolIdAttributeReference = AttributeReference.apply("SchoolID", LongType)(ExprId.apply(1))
   private val schoolStartDateAttributeReference = AttributeReference.apply("StartDate", DateType)(ExprId.apply(2))
   private val fields = List(AttributeReference.apply("SchoolID", LongType)(ExprId.apply(1), List("SUBQUERY_2")))
+  private val expressionFactory: SparkExpressionFactory = new SparkExpressionFactory {
+    override def createAlias(child: Expression, name: String, exprId: ExprId, qualifier: Seq[String], explicitMetadata: Option[Metadata]): Alias = {
+      Alias(child, name)(exprId, qualifier, explicitMetadata)
+    }
+  }
+  @Mock
+  var directBigQueryRelationMock: DirectBigQueryRelation = _
+  @Mock
+  var sparkPlanFactoryMock: SparkPlanFactory = _
```

Comment:
Why do we need to define `sparkPlanFactoryMock` here? Can't we use the one defined in the TestConstants class?

Reply:
Addressed